commit f64dbe1ddb4b16f2efdfac732c2d3b6e491a7736
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Tue Oct 30 12:23:26 2018 +0000

    config: Add bcm2835-codec to Pi defconfigs.
    
    Adds the V4L2 M2M codec driver to the config.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index f59e104f817c..556cb0fcaec7 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -1210,6 +1210,7 @@ CONFIG_FB_TFT_FBTFT_DEVICE=m
 CONFIG_BCM2835_VCHIQ=y
 CONFIG_SND_BCM2835=m
 CONFIG_VIDEO_BCM2835=m
+CONFIG_VIDEO_CODEC_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 507aa4685b35..c58c3a5fb1a9 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -1203,6 +1203,7 @@ CONFIG_FB_TFT_FBTFT_DEVICE=m
 CONFIG_BCM2835_VCHIQ=y
 CONFIG_SND_BCM2835=m
 CONFIG_VIDEO_BCM2835=m
+CONFIG_VIDEO_CODEC_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1386_f64dbe1ddb4b16f2efdfac732c2d3b6e491a7736.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1386_f64dbe1ddb4b16f2efdfac732c2d3b6e491a7736.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
